shifthelper/checks.py

```diff
@@ -1,53 +1,84 @@
 from custos import IntervalCheck
-import smart_fact_crawler
+import smart_fact_crawler as sfc
 from .tools.is_shift import is_shift_at_the_moment
+import requests
+from abc import ABCMeta, abstractmethod
```

Comment:
I learned, that `abstractmethod` is suitable when the API designer wants to express: "this thing is utterly useless without implementing this method".
Also it prevents from typos...